controllers/homeRoutes.js

```diff
@@ -64,16 +65,28 @@ router.get('/space/:space_id/access', withNoMembership, withAuth, async (req, re
     res.status(400).json(err);
     console.log(err);
   }
+});
+
+// Get idea create page
+router.get('/space/:space_id/ideas', withAuth, async (req, res) => {
+  try {
+    const spaceData = await Space.findByPk(req.params.space_id);
+    const space = spaceData.toJSON();
 
+    res.render('ideaCreate', { space });
+  } catch (err) {
+    res.status(400).json(err);
+    console.log(err);
+  }
 });
 
-// Create idea page
-router.get('/space/:space_id/idea', withAuth);
 
 // View a specific idea
-router.get('/idea/:id', withAuth, async (req, res) => {
+router.get('/space/:space_id/idea/:idea_id', withAuth, async (req, res) => {
```

Comment:
Add the `withApprovedMembership` authentication helper. See `/space/:space_id` route

Reply:
Done :)